jupyter_notebook_parser/parser.py

```diff
@@ -1,87 +1,103 @@
 import json
-from typing import List, Dict
+from os.path import splitext
 
 from jupyter_notebook_parser.container import SourceCodeContainer
 
 
 class JupyterNotebookParser:
-    def __init__(self, notebook_filename: str) -> None:
-        if not notebook_filename.endswith('.ipynb'):
-            raise NameError('This file is not a Jupyter notebook')
+    __notebook_corrupted_msg = "This notebook file is possibly corrupted"
```

Comment:
What's the purpose of using double leading underscores here?

Reply:
Double underscore marks 'private' function. His name will be mangled to prevent easy access from outside of class. Single undercore usualy used for 'protected' or hidden functions

Reply:
Single underscore means "private".  Double underscores have other meanings.